src/common/exceptions/generic-server.exception.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpException } from '../interfaces';
+
+export class GenericServerException implements HttpException {
+  readonly name: string = 'GenericServerException';
+  readonly message: string =
+    'The request could not be completed due to an internal server error.';
```

Comment:
might just say `The request could not be completed`

Reply:
Good catch.